src/binding/python/Attributable.cpp

```diff
@@ -365,7 +365,7 @@ void init_Attributable(py::module &m) {
                 return "<openPMD.Attributable with '" + std::to_string(attr.numAttributes()) + "' attributes>";
             }
         )
-        .def("series_flush", &Attributable::seriesFlush)
+        .def("series_flush", (void (Attributable::*)()) &Attributable::seriesFlush)
```

Comment:
Do you try to select the non-argument overload? :)
```suggestion
        .def("series_flush",
             py::overload_cast< >(&Attributable::seriesFlush))
```
https://pybind11.readthedocs.io/en/stable/classes.html#overloaded-methods

Reply:
Huh, didn't know that one.

Reply:
Jau, saves some C style function cast magic :)